specification/network/resource-manager/Microsoft.Network/stable/2018-01-01/examples/ApplicationGatewayBackendHealthGet.json

```diff
@@ -0,0 +1,73 @@
+{
+  "parameters": {
+    "api-version": "2018-01-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "appgw",
+    "applicationGatewayName": "appgw"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
```

Comment:
There is no property called `properties` in the definition of `ApplicationGatewayBackendHealth`. That means either the Swagger or this example is wrong ;) I assume your service does not wrap the entire response in a property called `properties`, i.e. you probably intended to have `backendAddressPools` here directly. 🙂